topologic/graph_augmentation.py

```diff
@@ -85,7 +85,7 @@ def rank_edges(
     """
     assertions.assert_is_weighted_graph(graph, weight_column)
 
-    edge_count = len(graph.edges)
+    edge_count = len(graph.edges())
```

Comment:
Can you do a grep on all instances of calls to `graph.edges` and `graph.nodes` and `g.edges` and `g.nodes` and see if there are any other cases we should proactively update to be method calls?